openhtf/util/xmlrpcutil.py

```diff
@@ -14,32 +14,40 @@
 
 """Utility helpers for xmlrpclib."""
 
-import httplib
-import SimpleXMLRPCServer
-import SocketServer
+import http.client
+import xmlrpc.server
+import os
+import socketserver
+import sys
 import threading
-import xmlrpclib
+import xmlrpc.client
+import collections
 
 DEFAULT_PROXY_TIMEOUT_S = 3
 
+if sys.version_info[0] < 3:
```

Comment:
Maybe a comment here about why this is necessary and the bugs in six that prevent us from using it?